src/chromium/cargo.md

```diff
@@ -35,6 +35,12 @@ Discuss in small groups the policies within your own team and organization,
 and come to a group agreement about what's an acceptable level of risk.
 
 <details>
+Talk about the cultural differences between the `cargo` world and the Chromium
```

Comment:
A quite subtle detail: please add a newline after the HTML. Without it, we won't extract the text for translation. Less importantly, `mdbook` won't process things like curly "typographic" quotes in the text that follows (which turns `---` into `—`).

```suggestion
<details>

Talk about the cultural differences between the `cargo` world and the Chromium
```